build.properties

```diff
@@ -1,2 +1,2 @@
 DOCKER_IMAGE_REPOSITORY=alfresco-base-tomcat
-DOCKER_IMAGE_TAG=0.1.0
+DOCKER_IMAGE_TAG=8.5.23-java-8u161-oracle-centos-7.4
```

Comment:
I think we said we only needed the feature version of the layers being consumed, i.e. `8.5.23-java-8-oracle-centos-7.4` ?

Reply:
ADR 0007 says it should be "<artifact-version>-<optional flavor>-<dependency>-<dependency flavor/version>....<dependency>-<dependency flavor/version>"

Ah, I had dependency flavor/version: `8u161-oracle` but you're right it should be `8-oracle`.

Reply:
@rgauss I think it should be `centos-7` *not* `centos-7.x`. The `.x` is irrelevant to an end administrator.